core/storage/question/gae_models_test.py

```diff
@@ -400,6 +400,24 @@ def test_get_more_question_skill_links_than_available(self):
         self.assertEqual(question_skill_links[2].skill_id, 'skill_id2')
         self.assertEqual(question_skill_links[2].question_id, 'question_id3')
 
+    def test_get_question_skill_links_based_on_mastery_invalid_mastery(self):
```

Comment:
As mentioned, we shouldn't be throwing such exceptions. We need to be setting degree_of_mastery = 0 for such cases. Change the tests to reflect that!

Reply:
Done.